app/routes/tasks.js

```diff
@@ -5,11 +5,21 @@ const API_BASE_URL = ENV.BASE_API_URL;
 
 export default class TasksRoute extends Route {
   model = async () => {
-    const response = await fetch(`${API_BASE_URL}/tasks/self`, {
+    await fetch(`${API_BASE_URL}/tasks/self`, {
       credentials: 'include',
-    });
-    const allTasks = await response.json();
-
-    return allTasks;
+    })
+      .then(function (response) {
+        if (!response.ok) {
+          throw Error(response.statusText);
+        }
+        return response.json();
+      })
+      .then(function (response) {
+        console.log('ok', response);
```

Comment:
We don't keep any console logs in production :)
They're fine for development purposes, but they should not go on prod